edudoc/staff-crash-course/staff_crash_course.md

```diff
@@ -0,0 +1,94 @@
+
+  - [The Code of Ethics](https://www.worldcubeassociation.org/documents/Code%20of%20Ethics.pdf); This describes how all WCA Staff must act and what they must abide by.
+  - [The Spirit of the WCA](https://www.worldcubeassociation.org/documents/motions/01.2019.1%20-%20Spirit.pdf).
+  - If you are a member of a committee or team; [The Committees and Teams Motion](https://www.worldcubeassociation.org/documents/motions/10.2019.0%20-%20Committees%20and%20Teams.pdf). This will explain how committees and teams operate. Additionally, you will find the relevant motion for your committee or team amongst the other listed motions on the documents page.
+  - If you are a Delegate; [The Delegates Motion](https://www.worldcubeassociation.org/documents/motions/08.2019.1%20-%20Delegates.pdf). This will explain how your work as a Delegate will be conducted. There is also a specific Crash Course for Delegates that you should familiarize yourself with.
+
+## Email
+
+The primary form of communication for the WCA is emails. All channels are managed by a platform called Google Groups; you should have been added to all relevant groups already as these are handled automatically when you are officially added to the team on the WCA website.\
+**When you respond to an email, use the “Reply All” option.** This is to ensure that all relevant people on the thread are still getting the reply.\
+Whatever mail interface you’re using, go and find the “Reply All” button. It will become your best friend. In particular, it helps to keep the team’s GGroup in a Carbon Copy (CC). Some interfaces (e.g. GMail) also allow setting “Reply All” as default reply option, which can be very helpful.
+
+When there are important matters within a region, or regarding a specific Delegate, you should CC the Senior Delegate.
+
+Please remember that the expected response time for WCA is 48 hours. If you are having trouble meeting this response time, please inform the recipient that you are expecting to give a delayed reply.\
+Additionally, if you have not received a reply in a timely manner, you should ping the corresponding party. While it is the responsibility of the corresponding party to take the appropriate action, sometimes you will need to gently remind them to do so.
+
+More broadly speaking, the preferred  approach is “over-communication” in the WCA by keeping all relevant persons in the loop at all times. If you think there is a topic which other staff members should see, please add them to as a Carbon Copy (CC) to the email. For example, if you escalate an issue about a specific Delegate to any Team/Committee, be sure to include their Senior and/or Regional Delegate in CC.
+
+<div style="page-break-after: always; visibility: hidden"> \pagebreak </div>
```

Comment:
I think it would be nicer to have a dedicated CSS class which does this, and just include something like
```md
\pagebreak
{.page-break}
```

I haven't tried it yet though, but given we use this syntax elsewhere I'd guess it works.

Reply:
Resolving myself, as it's indeed possible to attach attribute to headers/images, which are our only use cases.